R/expr-is.R

```diff
@@ -31,8 +31,12 @@ is_while_expr <- function(pd) {
   pd$token[1L] == "WHILE"
 }
 
+
+#' Check whether `pd` is a function call
 #' @describeIn pd_is Checks whether `pd` is a function call.
-#' @keywords internal
+#' @keywords other
```

Comment:
I don'nt think we need the `other` keyword do we?

Reply:
Probably not...